src/bento/aboutPageData.js

```diff
@@ -62,6 +62,11 @@ export const aboutData = {
   eleventhParagraph:
     "<a className='sectionTitle'>caDSR </a> <a className='aboutLink' href='https://cadsr.cancer.gov/onedata/Home.jsp' target='_blank' rel='noopener noreferrer'>https://cadsr.cancer.gov/onedata/Home.jsp</a><br/>" +
     "The Cancer Data Standards Repository is one of the largest CDE registries developed by the National Cancer Institute (NCI) and contains over 60,000 CDEs covering many aspects of cancer research. " ,
+  twelfthParagraph:
+    "For more information about the C3DC site, please refer to the "+
+    "<a className='aboutLink' href='https://github.com/CBIIT/c3dc-releases' target='_blank' rel='noopener noreferrer'>Release Notes</a> " +
```

Comment:
@huuaho Can you replace current link for Release Notes with "https://github.com/CBIIT/c3dc-releases/blob/main/README.md"?

Reply:
done